modules/configuring-oidc-authentication.adoc

```diff
@@ -0,0 +1,236 @@
+* *Verified Email Address Claim:* The name of the claim that is used to verify the email address of the user.
+* *Login Scopes:* The scopes to send to the OIDC provider when performing the login flow, for example, `openid`. After configuration, you must click *Add*. 
+
+. Scroll down and click *Validate Configuration Changes*. Then, click *Restart Now* to deploy the {productname} Operator with OIDC enabled. 
+
+
+[id="configuring-azuread-oidc"]
+== Configuring Azure AD OIDC for {productname}
+
+By integrating Azure AD authentication with {productname}, your organization can take advantage of the centralized user management and security features offered by Azure AD. Some features include the ability to manage user access to {productname} repositories based on their Azure AD roles and permissions, and the ability to enable multi-factor authentication and other security features provided by Azure AD.
+
+Azure Active Directory (Azure AD) authentication for {productname} allows users to authenticate and access {productname} using their Azure AD credentials.
+
+
+[id="configuring-azuread-using-config-tool"]
+=== Configuring Azure AD by using the {productname} config tool
+
+The following procedure configures Azure AD for {productname} using the config tool.
+
+.Procedure
+
+. Enter the {productname} config editor tool.
+
+.. If you are running a standalone {productname} deployment, you can enter the following command:
++
+[subs="verbatim,attributes"]
+----
+$ sudo podman run --rm -it --name quay_config -p 80:8080 -p 443:8443 {productrepo}/{quayimage}:{productminv} config secret
+----
++
+Use your browser to navigate to the user interface for the configuration tool and log in.
+
+.. If you are on the {productname} Operator, navigate to *Operators* -> *Installed Operators*. Click *Red Hat Quay* -> *Quay Registry*. Then, click the name of your {productname} registry, and the URL listed with *Config Editor Endpoint*.
+
+. Scroll down to the *External Authorization (OAuth)* section.
+
+. Click *Add OIDC Provider*.
+
+. When prompted, enter the ID for the ODIC provider.
++
+[NOTE]
+====
+Your OIDC server must end with `/`.
+====
+
+. After the ODIC provider has been added, {productname} lists three callback URLs that must be registered on Azure. These addresses allow Azure to direct back to {productname} after authentication is confirmed. For example:
+
+* `\https://QUAY_HOSTNAME/oauth2/<name_of_service>/callback`
+* `\https://QUAY_HOSTNAME/oauth2/<name_of_service>/callback/attach`
+* `\https://QUAY_HOSTNAME/oauth2/<name_of_service>/callback/cli`
+
+. After all required fields have been set, validate your settings by clicking *Validate Configuration Changes*. If any errors are reported, continue editing your configuration until the settings are valid and {productname} can connect to your database and Redis servers.
+
+////
+
+[id="configuring-azuread-updating-config-yaml"]
```

Comment:
@stevsmit why is commented out?

No matter if it is either an specific OIDC provider like Azure, like the one commented here, or a generic one, I think we should have a least a section and/or example of how to configure an ODIC provider using the config,yaml file.